examples/src/main/java/org/reactivestreams/example/lazycast/RangePublisher.java

```diff
@@ -0,0 +1,242 @@
+/************************************************************************
+ * Licensed under Public Domain (CC0)                                    *
+ *                                                                       *
+ * To the extent possible under law, the person who associated CC0 with  *
+ * this code has waived all copyright and related or neighboring         *
+ * rights to this code.                                                  *
+ *                                                                       *
+ * You should have received a copy of the CC0 legalcode along with this  *
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.*
+ ************************************************************************/
+
+package org.reactivestreams.example.lazycast;
```

Comment:
lazycast?

Reply:
I don't know the terminology in RS of what we call "cold observables" in RxJava.

Reply:
Also for us, `unicast` means a maximum of 1 `Subscriber` and `multicast` is associated with `Processor`-like behavior.

Reply:
I'm open to naming suggestion.

Reply:
`unicast` would make most sense, since it is a sync version of the async? https://github.com/reactive-streams/reactive-streams-jvm/blob/master/examples/src/main/java/org/reactivestreams/example/unicast/AsyncIterablePublisher.java

Reply:
Okay, I moved them there.